docs/en/reference/working-with-associations.rst

```diff
@@ -488,30 +471,52 @@ and the "remove" operation.
         //...
     }
 
-Since ``cascade: persist`` is configured for the ``User#commentsAuthored``
-association, you can now create a user and persist their comments as follows:
+...you can now create a user and an associated comment like this:
 
 .. code-block:: php
 
     <?php
     $user = new User();
-    $myFirstComment = new Comment();
-    $user->addComment($myFirstComment);
-    $myFirstComment->setUser($user);
+    $user->comment('Lorem ipsum', new DateTime());
     
     $em->persist($user);
     $em->flush();
 
+Thanks to ``cascade: remove``, you can easily delete a user too (without having to loop through all associated comments):
```

Comment:
OK, done.  
I kept the "Thanks" part, to indicate that the remove-cascading has in fact alread been set in the example above.